src/mergeWithDotPath.js

```diff
@@ -17,9 +17,10 @@ import assocDotPath from './assocDotPath';
  *
  *      R_.mergeWithDotPath('a.b', R.merge, { d: 30 }, {a: {b: { c: 20 }}}); //=> {a: {b: { c: 20, d: 30 }}}
  */
-const mergeWithDotPath = curry((path, mergeFn, value, obj) => assocDotPath(
-	path,
-	mergeFn(dotPath(path, obj), value), obj)
-);
+const takeFirst = nthArg(0);
+const takeForth = nthArg(3);
```

Comment:
- call it `lastArg` 
- implement it like following: `nthArg(-1)`
- and make it standalone function